packaging/README.release.md

```diff
@@ -59,15 +59,21 @@ gpg2 --armor --detach-sign dist/nmstate-<version>.tar.gz
 * Choose the git tag just pushed.
 * Title should be like `Version 0.0.3 release`.
 * The content should be copied from the `CHANGELOG` file.
-
-
- * Click `Attach binaries by dropping them here or selecting them.` and
-   upload the `dist/nmstate-<version>.tar.gz` and
-   `dist/nmstate-<version>.tar.gz.asc`.
-
- * Click `Save draft` and seek for review.
-
- * Click `Publish release` once approved.
+* Click `Attach binaries by dropping them here or selecting them.` and upload
+  the `dist/nmstate-<version>.tar.gz` and `dist/nmstate-<version>.tar.gz.asc`.
+* Download the tarball and the signature.
+* Check if the signature is correct.
+```bash
+gpg2 --recv-keys F7910D93CA83D77348595C0E899014C0463C12BB
+gpg --verify nmstate-<version>.tar.gz.asc nmstate-<version>.tar.gz
+```
+* Check in a clean container if the package build and install correctly.
+```bash
+python setup.py build
+python setup.py install
```

Comment:
Should there also be a short smoke test to run nmstatectl?

Reply:
AFAIK, this is only for testing the building and installation. I also ran the tests.. but after install all the dependencies.. thanks!